samples/ftd_configuration/physical_interface_with_default_dhcp.yml

```diff
@@ -0,0 +1,52 @@
+- hosts: all
+  connection: httpapi
+  vars:
+    iface_name: "inside"
+    iface_hardware_name: "GigabitEthernet0/1"
+  tasks:
+    - name: Get List of DHCP servers
+      ftd_configuration:
+        operation: "getDHCPServerContainerList"
+        register_as: dhcp_containers_list
+
+    - name: Execute 'editDHCPServerContainer' operation
```

Comment:
Let's rename it to smth more specific, e.g., `Remove default DHCP server`.

Reply:
renamed this step